### ISSUE TITLE
serialization error

### ISSUE THREAD

**Issue by tamasblummer** [defiads/defiads#11]
defiads panics if requested content from an other node with:
```
15:39:09 [ERROR] thread 'updater' panicked at 'called `Result::unwrap()` on an `Err` value: ErrorImpl { code: Message("invalid type: integer `3`, expected a bytestring"), offset: 0 }'
: src/libcore/result.rs:1084
```
Very likely a problem in CBOR serialization. Serialization needs to be re-worked either... Doing it.

**Comment by tamasblummer** [comment]
fixed in 760ae81ba3b6d223e73103038fc95c10ec0e233d

**Comment by tamasblummer** [comment]
fixed in 0.2.1
